etc/kayobe/ansible/reboot.yml

```diff
@@ -2,6 +2,12 @@
 - name: Reboot the host
   hosts: seed-hypervisor:seed:overcloud:infra-vms
   serial: "{{ lookup('env', 'ANSIBLE_SERIAL') | default(1, true) }}"
+  vars:
+    gather_facts: false
```

Comment:
This should be a top level play attribute, not a variable.

Reply:
done